.github/workflows/integration-test.yaml

```diff
@@ -0,0 +1,34 @@
+name: Integration Test Preview
+on:
+  deployment_status
+jobs:
+  integration-test:
+    name: Integration Test
+    runs-on: ubuntu-latest
+    if: github.event.deployment_status.environment == 'Preview' && github.event.deployment_status.creator.login == 'vercel[bot]'
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+
+      - name: Use Node v14
+        uses: actions/setup-node@v1
+        with:
+          node-version: '14.x'
+
+      - name: Install deps
+        run: yarn --frozen-lockfile --no-progress --ignore-scripts
+
+      # Install and check Cypress binary
+      - run: yarn run cypress install
```

Comment:
Let's circle back on this later and see if we can enable proper caching.

Reply:
Workflows triggered by a `deployment_status` event don't support caching yet. You can see the warning/error output in the pipeline [here](https://github.com/Basis-Theory-Labs/send-securely/runs/6374324477?check_suite_focus=true#step:4:7).